edx_proctoring/static/proctoring/templates/proctored-exam-info.underscore

```diff
@@ -10,7 +10,13 @@
     <%} %>
     <div class="onboarding-reminder">
     <% if (showOnboardingReminder) { %>
-        <h4 class="message-title"><%= gettext("You must complete the onboarding process prior to taking any proctored exam.") %></h4>
+        <h4 class="message-title">
+            <% if (showOnboardingExamLink) { %>
+                <%= gettext("You must have a verified onboarding profile prior to taking any proctored exam.") %>
+            <% } else { %>
+                <%= gettext("You have successfully submitted your onboarding profile for review.") %>
```

Comment:
This would theoretically show this message to users with an onboarding exam attempt status of `second_review_required`. That may be a source of confusion.

Reply:
I like how you noticed this. I want to use wording that make sense to both "Submitted" and "Second_review_required" learners. What sentence do you think make sense?
Maybe `Your submitted onboarding profile is in review.`?

Reply:
I like your suggestion. I think that is clearer!

Reply:
@MichaelRoytman Suggestion Taken